driver/makedepend.ml

```diff
@@ -409,7 +410,10 @@ let process_file_as process_fun def source_file =
   List.iter add_to_load_path (
       (!Compenv.last_include_dirs @
        !Clflags.include_dirs @
-       !Compenv.first_include_dirs
+       if not !nocwd then
+         [Filename.current_dir_name] @
+         !Compenv.first_include_dirs
+       else !Compenv.first_include_dirs
```

Comment:
The form `[x] @ l` is a complex way to write `x::l`. Moreover, besting the `if` in the list of paths to be included looks a bit noisy. If you prefer to not split the list iter, I believe it is better to define a subexpression:

```ocaml
let cwd = if !nocwd then [] else [Filename.current_dir_name] in
...
       !Compenv.last_include_dirs @
       !Clflags.include_dirs @
       !Compenv.first_include_dirs @
       cwd @
       !Compenv.first_include_dirs
```

Reply:
why are we adding `!Compenv.first_include_dirs` twice ?

Reply:
Once is enough indeed.